Update Create External Secrets  Docs (#3847)
weaveworks/weave-gitops: website/docs/secrets/manage-secrets-ui.mdx

It's important to note that the chosen `SecretStore` may be a cluster-scoped `SecretStore` ie: `ClusterSecretStore` or a namespace-scoped `SecretStore`.

> If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.

Reviewer: what is the scenarios we have in mind when having both `secretstore` and `secret` in the same namespace? 

my basic understanding tells me that secretstore is being provisioned as part of the operator so it is a platform capability owned by platform team. however, the external secret is the manifest to allow other teams to use the capability. having both of them in the same namespace might involve that both platform teams and application team have access to the same namespace that might not be the case.

Author: yes you are rigth, if the team who is responsible for the secretStore creation diffrent from the teams who will add and sync thier ExternalSecrets and they don't have the access to the same namespace as platform teams so they need to use a Cluster scoped SecretStore https://external-secrets.io/v0.4.2/api-clustersecretstore/ which will be availble and can be refrenced from any namespace.

![Create new Secret](./imgs/create-secret-2.png)

If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.

![Create new Secret](./imgs/create-secret-3.png)

#### Adding Secret Properties

> After you have choose your desired `SecretStore` & `SecretPath` the UI allows you to add secret properties in two different scenarios:

Reviewer: ```suggestion
After you have chosen your desired `SecretStore` & `SecretPath` the UI allows you to add secret properties in two different scenarios:
```

Reviewer: Re `SecretPath` What is the information that the user needs before populating this field? I was looking at whether that info was in the UI or in the docs. I found it a bit later in the doc https://docs.gitops.weave.works/docs/secrets/manage-secrets-ui/#external-secret-details 

A user first facing the doc might require them to understand either from us or from other links.  It would be cool if they could be referenced as they first time appear or similar.

##### 1. Adding and Removing Multiple Property Fields

The first scenario allows you to add or remove multiple property fields. Each added `property` also has an optional `SecretKey` field. Here's how to do it:

In the `Properties` section, click the `Add` button to create a new property field.

![Add Property](./imgs/add-property-1.png)

Enter the name of the `property` you want to create. You can add as many properties as you need.

![Enter Property](./imgs/enter-property-1.png)

If you wish to specify a `SecretKey` for the property, enter it in the `SecretKey` field. If this field is left blank, the `property` name will be used as the `SecretKey`.

> [Add SecretKey](./imgs/add-secret-key-1.png)

Reviewer: ```suggestion
![Add SecretKey](./imgs/add-secret-key-1.png)
```

> 

Reviewer: It could be good to extend the piece of how the example translates as `Kubernetes Secret` to provide the complete example or a link for that example (for example to original ESO website)

To remove a property, click the `Remove` sign next to the property you wish to delete.

Remember, this option allows you to have fine-grained control over which properties are included in your `ExternalSecret`.

##### 2. Including All Properties

The second scenario is to include all properties in your `ExternalSecret`. If the `Include all properties` checkbox is checked, all property inputs will be disabled and ignored, and all secrets including all keys under the specified `SecretPath` will be added. Here's how:

Check the `Include all properties` checkbox. This will automatically disable the property input fields.

![Include All Properties Checkbox](./imgs/include-all-properties-1.png)

Using this option allows you to quickly create an `ExternalSecret` that includes all secrets under a specific `SecretPath`, without the need to specify each one individually. 

> Remember to use this option with caution.

Reviewer: This is a great call out. It would be good to extend with the reasoning behind this caution for the user to understand the risks.  Given that is a warning, it might be good to use some https://docusaurus.io/docs/markdown-features/admonitions

![Enter Property](./imgs/enter-property-1.png)

If you wish to specify a `SecretKey` for the property, enter it in the `SecretKey` field. If this field is left blank, the `property` name will be used as the `SecretKey`.

![Add SecretKey](./imgs/add-secret-key-1.png)

To remove a property, click the `Remove` sign next to the property you wish to delete.

Remember, this option allows you to have fine-grained control over which properties are included in your `ExternalSecret`.

##### 2. Including All Properties

The second scenario is to include all properties in your `ExternalSecret`. If the `Include all properties` checkbox is checked, all property inputs will be disabled and ignored, and all secrets including all keys under the specified `SecretPath` will be added. Here's how:

Check the `Include all properties` checkbox. This will automatically disable the property input fields.

![Include All Properties Checkbox](./imgs/include-all-properties-1.png)

Using this option allows you to quickly create an `ExternalSecret` that includes all secrets under a specific `SecretPath`, without the need to specify each one individually.

:::caution

> Remember to use this option with caution. You may not need to expose all your ExternalSecrets to be on the cluster.

Reviewer: ```suggestion
Remember to use this option with caution. You may not need to expose all your secret properties to be on the cluster.
```

- **Secret Store:** This shows the name of the secret store provider that is being used to store the external secret.
- **Secret Store Type:** This indicates the type of secret store that is being used to store the external secret. In this case, the type is "AWS Secrets Manager".
- **Secret path:** This is the path to the external secret within the secret store.
- **Version:** This shows the version of the external secret, which may be blank if no version has been specified.

- #### Properties

  Based on the configuration of the external secret, this section will vary:

> - If the "Include all properties" option was selected during the creation of the external secret, this section will display the text "All properties are included".

Reviewer: Please take a look at the screenshoot: it seems to be test data for development of the feature. Given that this is end-user documentation, it might be better to have pictures that would reference some realistic scenario to help improve user understanding.
